### PR TITLE
the header sits outside the msk site wrapper component

### PR DESCRIPTION
**Pull request by AncaComanescu**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,35 +11,35 @@ import '@knapsack-cloud/msk-design-system/dist/main.css';
...
-            </Route>
-          </Switch>
-        </Router> 
-        <MskFooter
-          links={footerLinks}
-        />
-      </div>
-    </MskSiteWrapper>
+      <MskSiteWrapper>
+        <div>
```
Comment (bronzehedwick) [inline src/App.js]: Do we need this extra div wrapper?

File: src/App.js [hunk 2]
```diff
@@ -11,35 +11,35 @@ import '@knapsack-cloud/msk-design-system/dist/main.css';
...
 function App() {
-  let footerLinks=[
+  let footerLinks = [
     { text: 'Example', href: 'https://example.com' },
     { text: 'Hello, world', href: 'https://helloworld.com' },
     { text: 'FooBar', href: 'https://foobar.com' },
   ];
   return (
-    <MskSiteWrapper>
+    <div>
```
Comment (bronzehedwick) [inline src/App.js:20]: Should this wrapper div be changed to a JSX fragment? aka `<></>`? Just want to reduce as much extraneous markup as possible to have the best test of the components.

**Review by bronzehedwick — CHANGES_REQUESTED** [review]
Just a few questions. Thanks!

File: src/App.js (hunk 1 above)
Comment (AncaComanescu) [reply src/App.js]: I removed the extra div element.

**Review by bronzehedwick — APPROVED** [review]

**Pull request merged**